docs/get-started/mapbox-tokens.md

```diff
@@ -18,22 +18,24 @@ We recommend using an environment variable to minimize leaking risks. See [secur
 
 ## Display Maps Without A Mapbox Token
 
-It is possible to use the map component without the Mapbox service, if you use another tile source (for example, if you host your own map tiles). Note that this is no longer allowed using mapbox-gl v2.0 and above. The options are:
+It is possible to use the map component without the Mapbox service, if you use another tile source (for example, if you host your own map tiles). Note that this is no longer allowed using `mapbox-gl` v2.0 and above. The options are:
 
+- Use [maplibre-gl](https://maplibre.org/maplibre-gl-js-docs/api/). See [Get Started](./get-started.md) for how to configure your project.
 - Stay on `mapbox-gl@1.x`. react-map-gl plans to continue supporting this release in the foreseeable future, however, this version will not include any of the latest features of the map renderer, nor get any future updates from Mapbox.
-- Use a community fork of mapbox-gl, for example [maplibre-gl](https://www.npmjs.com/package/maplibre-gl). See [Get Started](./get-started.md) for how to configure your project.
 
-To use your own map service, you will need a custom Mapbox GL style that points to your own [vector tile source](https://www.mapbox.com/mapbox-gl-js/style-spec/), and pass it to `Map` using the `mapStyle` prop. This custom style must match the schema of your tile source.
+To use your own map service, you will need a custom map style that points to your own [tile source](https://maplibre.org/maplibre-style-spec/), and pass it to `Map` using the `mapStyle` prop.
 
-Open source tile schemas include:
+Some alternative map data providers include:
 
-- [TileZen schema](https://tilezen.readthedocs.io/en/latest/layers/)
-- [OpenMapTiles schema ](https://openmaptiles.org/schema/)
+- [MapTiler](https://www.maptiler.com)
+- [Amazon Location Service](https://docs.aws.amazon.com/location/latest/developerguide/map-concepts.html)
 
-Some useful resources for creating your own map service:
+Useful resources for creating your own map tiles and service:
```

Comment:
Pmtiles is a great and easy way to host tiles using a single zip file.
It needs the relevant plugin for maplibre though.